crates/router/src/connector/stripe.rs

```diff
@@ -1693,7 +1699,7 @@ impl api::IncomingWebhook for Stripe {
         &self,
         request: &api::IncomingWebhookRequestDetails<'_>,
     ) -> CustomResult<api::IncomingWebhookEvent, errors::ConnectorError> {
-        let details: stripe::WebhookEvent = request
+        let details: stripe::WebhookEventTypeBody = request
             .body
             .parse_struct("WebhookEvent")
             .change_context(errors::ConnectorError::WebhookReferenceIdNotFound)?;
```

Comment:
Is this error variant correct?

Reply:
No. A more appropriate variant would be `EventTypeNotFound`.  I'll create a separate PR for this.